cypress/support/commands.js

```diff
@@ -23,3 +23,11 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder="${placeholder}"]`);
+});
+
+Cypress.Commands.add('findByType', (type) => {
+  cy.get(`[type="${type}"]`);
+})
\ No newline at end of file
```

Comment:
Add a newline at the end of the file for better readability and to follow general coding standards. 😊